default/scripting/common/misc.macros

```diff
@@ -40,6 +40,16 @@ SYSTEM_MINES_DAMAGE_FACTOR
 SUPPLY_DISCONNECTED_INFLUENCE_MALUS
 '''1'''
 
+MINIMUM_DISTANCE_CAPITAL_CHECK
```

Comment:
Seems misnamed. Is not specifically capital related

Reply:
Ophi mentioned that too. Changed to EMPIRE_CHECK